packages/flutter/lib/src/material/tabs.dart

```diff
@@ -1122,10 +1122,13 @@ class _TabBarViewState extends State<TabBarView> {
       _controller.animation.addListener(_handleTabControllerAnimationTick);
   }
 
+  List<Widget> ensureGlobalKeysForList(List<Widget> items) =>
+      List<Widget>.generate(items.length, (int index) => KeyedSubtree(key: GlobalKey(), child: items[index]));
```

Comment:
formatting nit: It's usually easier to read to format Widgets so that there's one argument per line:

```dart
KeyedSubtree(
  key: GlobalKey(), 
  child: items[index],
)
```